users/src/main/java/io/crate/operation/user/TransportPrivilegesAction.java

```diff
@@ -127,11 +127,24 @@ private static long applyPrivilege(Set<Privilege> privileges, Privilege privileg
             case GRANT:
                 privileges.add(privilege);
                 return 1L;
-            case REVOKE:
+            case DENY:
                 Privilege grantPrivilege = Privilege.privilegeAsGrant(privilege);
```

Comment:
wouldnt it be easier to have a map here as privileges using a comparator comparing the object? this would save you from creating lookup objects

revoke would be a lot easier too